content/millennium/r4/base/individuals/related-person.md

```diff
@@ -214,15 +214,14 @@ The common [errors] and [OperationOutcomes] may be returned.
 
 ## Patch
 
-Patch an existing Encounter-level RelatedPerson.
+Patch an existing Encounter-level or Person-level RelatedPerson.
```

Comment:
Person-level should change to Patient-level. I apologize for having that incorrect on the JIRA. I went ahead and updated that on the JIRA to Patient-level.

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/686/commits/31e9db51448df7fb3b00319bcefbface2ce5343c#diff-2e663b902e2663663b5f937e76fc0abb32406c03fbea8ee6d4dc4a7f6eb56584R217

Reply:
Thank you.